Automattic/wp-calypso
# Jetpack Pricing Page: Remove daily products

#### Changes proposed in this Pull Request

This PR removes the daily products from the pricing page, and instead only displays real-time products.

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/42627630/120542629-ef37bd00-c3b0-11eb-81da-08e189c92015.png">

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/42627630/120542669-fb237f00-c3b0-11eb-8de4-551fc9c3e5b8.png">

#### Testing instructions



1. Visit cloud.jetpack.localhost:3001/pricing.
2. Verify that daily products do not display, and that real time products display instead.
